### PR TITLE
replace forks: chatkit, fresco, popupBubble

### LINKED ISSUE #1197: Patched ChatKit Implementation missing
see gradle file

```
implementation('com.github.mario:chatkit:c6a61767291ddb212a2f4f792a2b6aaf295e69a5', {
        exclude group: 'com.facebook.fresco'
})
```

@mario Any chance you have a backup somewhere which you could push so we can fork/transfer it? Or remember what you pathed? ❤️

### PR DESCRIPTION
**Pull request by mahibi**
something is still wrong here.
complains about wrong dependencies.

eg. something with "com.stfalcon.chatkit"

will have a look later

Fix #1197

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
@mahibi for popupBubble we could replace it with the original repo/master-snapshot which has the patch too. Or in the midterm solution space replace it by a material button and remove the dependency completely

**Comment by mahibi** [comment]
libs moved to nextcloud org! PR must be updated to use the libs from there.
for a first step, the exact commit's as before might be used.

File: app/build.gradle [hunk 1]
```diff
@@ -238,15 +240,15 @@ dependencies {
     implementation 'org.apache.commons:commons-lang3:3.12.0'
     implementation 'com.github.wooplr:Spotlight:1.3'
     implementation 'com.google.code.findbugs:jsr305:3.0.2'
-    implementation('com.github.mario:chatkit:c6a61767291ddb212a2f4f792a2b6aaf295e69a5', {
+    implementation('com.github.mahibi:ChatKit:master-SNAPSHOT', {
```
Comment (nickvergessen) [inline app/build.gradle:243]:
```suggestion
    implementation('com.github.nextcloud:ChatKit:master-SNAPSHOT', {
```

File: app/build.gradle [hunk 2]
```diff
@@ -238,15 +240,15 @@ dependencies {
...
-    implementation 'com.github.mario.fresco:fresco:111'
-    implementation 'com.github.mario.fresco:animated-webp:111'
-    implementation 'com.github.mario.fresco:webpsupport:111'
-    implementation 'com.github.mario.fresco:animated-gif:111'
-    implementation 'com.github.mario.fresco:imagepipeline-okhttp3:111'
+    implementation 'com.github.mahibi.fresco:fresco:111'
+    implementation 'com.github.mahibi.fresco:animated-webp:111'
+    implementation 'com.github.mahibi.fresco:webpsupport:111'
+    implementation 'com.github.mahibi.fresco:animated-gif:111'
+    implementation 'com.github.mahibi.fresco:imagepipeline-okhttp3:111'
```
Comment (nickvergessen) [inline app/build.gradle:251]:
```suggestion
    implementation 'com.github.nextcloud.fresco:fresco:111'
    implementation 'com.github.nextcloud.fresco:animated-webp:111'
    implementation 'com.github.nextcloud.fresco:webpsupport:111'
    implementation 'com.github.nextcloud.fresco:animated-gif:111'
    implementation 'com.github.nextcloud.fresco:imagepipeline-okhttp3:111'
```

File: app/build.gradle [hunk 3]
```diff
@@ -258,7 +260,7 @@ dependencies {
     implementation 'com.novoda:merlin:1.2.1'
 
     implementation 'com.github.Kennyc1012:BottomSheet:2.4.1'
-    implementation 'com.github.mario:PopupBubble:a365177d96'
+    implementation 'com.github.mahibi:PopupBubble:master-SNAPSHOT'
```
Comment (nickvergessen) [inline app/build.gradle:263]:
```suggestion
    implementation 'com.github.nextcloud:PopupBubble:master-SNAPSHOT'
```

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Review by AndyScherzinger — APPROVED** [review]
Changes applied, versions fixed, build 🟢 :)

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**